packages/perseus/src/widgets/label-image/__tests__/label-image.test.ts

```diff
@@ -709,4 +714,110 @@ describe("LabelImage", function () {
+                        },
+                        {
+                            label: "The second unlabeled bar line.",
+                            selected: undefined,
+                        },
+                        {
+                            label: "The first unlabeled bar line.",
+                            selected: undefined,
+                        },
+                    ],
+                },
+            });
+        });
+    });
+
+    describe("scorePerseusItem", () => {
+        it("should be invalid on first render", () => {
+            // Arrange
+            const {renderer} = renderQuestion(textQuestion);
+
+            // Act
+            const score = scorePerseusItemTesting(
+                textQuestion,
+                renderer.getUserInputMap(),
+            );
+
+            // Assert
+            expect(score).toHaveInvalidInput();
+        });
+
+        it("can be answered correctly when correct option is picked for the marker", async () => {
+            // Arrange
+            const {renderer} = renderQuestion(shortTextQuestion);
+
+            // Act
+            const markerButton = screen.getByRole("button", {
+                name: "The fourth unlabeled bar line.",
+            });
+            await userEvent.click(markerButton);
+
+            const choice = screen.getByRole("option", {name: "SUVs"});
+            await userEvent.click(choice);
+
+            const score = scorePerseusItemTesting(
+                textQuestion,
+                renderer.getUserInputMap(),
+            );
+
+            // Assert
+            expect(score).toHaveBeenAnsweredCorrectly();
+        });
+
+        it("can be answered incorrectly when incorrect option picked for the marker", async () => {
+            // Arrange
+            const {renderer} = renderQuestion(shortTextQuestion);
+
+            // Act
+            const markerButton = screen.getByRole("button", {
+                name: "The fourth unlabeled bar line.",
+            });
+            await userEvent.click(markerButton);
+
+            const choice = screen.getByRole("option", {name: "Trucks"});
+            await userEvent.click(choice);
+
+            const score = scorePerseusItemTesting(
+                textQuestion,
+                renderer.getUserInputMap(),
+            );
+
+            // Assert
+            expect(score).toHaveBeenAnsweredIncorrectly();
+        });
+    });
+});
+
+describe("textWithoutAnswersQuestion", () => {
+    it("should render a text question without answers", () => {
+        // @ts-expect-error - Type not assignable to PerseusRenderer (answers removed)
+        renderQuestion(textWithoutAnswersQuestion);
+
+        const text = screen.getByText(
+            "Carol created a chart and a bar graph to show how many of each type of vehicle were in her supermarket parking lot.",
+        );
+
+        // expect(container).toMatchSnapshot();
+        expect(text).toBeInTheDocument();
```

Comment:
This is closer but not quite what I was suggesting:

> - Render a LabelImage widget using ItemData that has no answers (to simulate how it will be on the FE)
> - Answer the LabelImage (both wrong and right)
> - Use getUserInput on the Renderer to get the user input
> - Use scorePerseusItem to score the user input

So you're rendering a LabelImage widget without answers, but only asserting that the content around the widget renders (the text you're getting isn't a part of the widget). What I'm proposing is that we repeat "can be answered correctly when correct option is picked for the marker" and "can be answered incorrectly when incorrect option picked for the marker" using `textWithoutAnswersQuestion`.

Reply:
We ended up deciding to handle rendering without answers with future related work. Kept this test to show that rendering doesn't rely on having answer data (the answer arrays are still in the test data, but they are empty. Can modify or remove this test when we start fully handling rendering with no answer property.